lib/xandra/cluster.ex

```diff
@@ -0,0 +1,96 @@
+defmodule Xandra.Cluster do
+  @behaviour DBConnection.Pool
+
+  alias __MODULE__.{ControlConnection, StatusChange}
+
+  defstruct [:options, active: %{}, idle: %{}]
```

Comment:
`:active` and `:idle` read pretty mysterious IMO. What about `:active_pools` and `:idle_pools`?